data/DataType/long_c.yaml

```diff
@@ -1,4 +1,4 @@
-keyword: int
```

Comment:
I think you meant to rename this , not delete it.

Reply:
Not sure what happened on GitHub, because I have renamed the file and it exists on my local repo.

![screenshot from 2018-11-09 21-11-00](https://user-images.githubusercontent.com/25429745/48272307-4d53cc00-e464-11e8-99e3-09201afa23af.png)

Reply:
Oh this is because you merged my earlier PR containing those int and short files.